db/migrate/20190715092943_create_versions.rb

```diff
@@ -9,7 +9,7 @@ class CreateVersions < ActiveRecord::Migration[5.2]
   TEXT_BYTES = 1_073_741_823
 
   def change
-    create_table :versions, { options: "ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_general_ci" } do |t|
```

Comment:
We need to enforce UTF-8, this is required by Rails 6. I think we should keep this.